parser/configurations_parser.go

```
@@ -127,6 +127,14 @@ func (bp configurationsParser) ReadConfigurationsServices() ([]BeelzebubServiceC
 		return nil, fmt.Errorf("in directory %s: %v", bp.configurationsServicesDirectory, err)
 	}
 
+	// function to test a smell code
+	naming_like_python := 12345
+	log.Infoln(naming_like_python)
+
+	for _, _ = range services {
```

golangci-lint (1.64.8)

S1005: unnecessary assignment to the blank identifier

(gosimple)